resources/lib/modules/system.py

```diff
@@ -564,11 +565,11 @@ def tar_add_folder(self, tar, folder):
                         self.tar_add_folder(tar, itempath)
                 else:
                     self.done_backup_size += os.path.getsize(itempath)
-                    log.log(f'Adding to backup: {itempath}', log.DEBUG)
+                    log.log(f'Adding to backup: {log.utf8ify(itempath)}', log.DEBUG)
                     tar.add(itempath)
                     if hasattr(self, 'backup_dlg'):
                         progress = round(1.0 * self.done_backup_size / self.total_backup_size * 100)
-                        self.backup_dlg.update(int(progress), f'{folder}')
+                        self.backup_dlg.update(int(progress), f'{print_folder}\n{log.utf8ify(item)}')
```

Comment:
`print_folder` can be eliminated to log.utf8ify(folder) here? I don't see it used elsewhere.

Reply:
The function call was moved out of the loop.